SocialCareCaseViewerApi/Startup.cs

```diff
@@ -116,22 +111,9 @@ public static void ConfigureServices(IServiceCollection services)
 
             services.AddHttpClient<ISocialCarePlatformAPIGateway, SocialCarePlatformAPIGateway>(client =>
             {
-                client.BaseAddress = new Uri(Environment.GetEnvironmentVariable("SOCIAL_CARE_PLATFORM_API_URL"));
+                client.BaseAddress = new Uri(Environment.GetEnvironmentVariable("SOCIAL_CARE_PLATFORM_API_URL") ?? throw new InvalidOperationException("Must provide SOCIAL_CARE_PLATFORM_API_URL environment variable"));
             });
 
-            services.AddTransient<IValidator<CreateAllocationRequest>, CreateAllocationRequestValidator>();
-            services.AddTransient<IValidator<UpdateAllocationRequest>, UpdateAllocationRequestValidator>();
-            services.AddTransient<IValidator<CreateWorkerRequest>, CreateWorkerRequestValidator>();
-            services.AddTransient<IValidator<PatchWarningNoteRequest>, PatchWarningNoteRequestValidator>();
-            services.AddTransient<IValidator<CreateTeamRequest>, CreateTeamRequestValidator>();
-            services.AddTransient<IValidator<GetTeamsRequest>, GetTeamsRequestValidator>();
-            services.AddTransient<IValidator<CreateCaseSubmissionRequest>, CreateCaseSubmissionRequestValidator>();
-            services.AddTransient<IValidator<UpdateCaseSubmissionRequest>, UpdateCaseSubmissionRequestValidator>();
-            services
-                .AddTransient<IValidator<UpdateFormSubmissionAnswersRequest>, UpdateFormSubmissionAnswersValidator>();
-            services.AddTransient<IValidator<CreatePersonalRelationshipRequest>, CreatePersonalRelationshipRequestValidator>();
-            services.AddTransient<IValidator<QueryCaseSubmissionsRequest>, QueryCaseSubmissionsValidator>();
-
```

Comment:
Why are we removing these? Do we no longer need the validators?

Reply:
I don't believe these are doing anything.

In our constructors we actually the validator and call it against the request there doesn't seem to be a requirement to inject the validator into the Startup class.